examples/example_emscripten_wgpu/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# Example usage:
+#  cmake -B build -g "Ninja"
+
+cmake_minimum_required(VERSION 3.10.2)
+project(imgui_example_emscripten_wgpu C CXX)
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Debug CACHE STRING "" FORCE)
+endif()
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DVK_PROTOTYPES")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DVK_PROTOTYPES")
+
+set(BUILD_TESTING OFF)
+set(BUILD_TESTING_STATIC  OFF)
+set(BUILD_TESTING_SHARED OFF)
+
+# Dear ImGui
+set(IMGUI_DIR ../../)
```

Comment:
Better do this:
```cmake
set(IMGUI_DIR "${CMAKE_CURRENT_LIST_DIR}/../..")
```
1. `CMAKE_CURRENT_LIST_DIR` usage says clearly what the path is relative to
2. The use of quotes makes it work for paths where `CMAKE_CURRENT_LIST_DIR` might be a path with spaces in it.

Reply:
Looks good, might want to do the same change in the vulkan example too.